homeassistant/components/proximity.py

```diff
@@ -12,17 +12,33 @@
 from homeassistant.helpers.entity import Entity
 from homeassistant.helpers.event import track_state_change
 from homeassistant.util.location import distance
+from homeassistant.util.distance import convert
+from homeassistant.const import ATTR_UNIT_OF_MEASUREMENT
 
 DEPENDENCIES = ['zone', 'device_tracker']
 
 DOMAIN = 'proximity'
 
+NOT_SET = 'not set'
+
 # Default tolerance
 DEFAULT_TOLERANCE = 1
 
 # Default zone
 DEFAULT_PROXIMITY_ZONE = 'home'
 
+# Default unit of measure
+DEFAULT_UNIT_OF_MEASUREMENT = 'km'
```

Comment:
I'm gonna go ahead and do that - seems like a larger change that shouldn't necessarily be a part of this PR